### PR TITLE
CAIM-22 Generate PDF from foster application

### PR DESCRIPTION
**Pull request by rwwiv**

### CODE REVIEW

File: caim/settings.py [hunk 1]
```diff
@@ -32,6 +32,9 @@
 DEBUG = os.getenv("DEBUG", "0") == "1"
 ALLOWED_HOSTS = ["*"]
 
+APP_DIR = os.path.join(BASE_DIR, "caim_base")
+STATIC_ROOT = os.getenv("STATIC_ROOT", os.path.join(APP_DIR, "static"))
```
Comment (rgasper) [inline caim/settings.py:36]: will this change require rerunning `python manage.py collectstatic` after deploy?

I'm not entirely familiar with how our staticfile hosting is handled in production

File: caim_base/views/fosterer_profile.py [hunk 2]
```diff
@@ -357,9 +367,60 @@ def edit(request, stage_id):
...
+            "form": form,
             "pageTitle": "Edit your fosterer profile",
             "stageNumber": stage_number,
         },
     )
+
+@login_required()
+def download_fosterer_profile(request: HttpRequest, fosterer_id: int) -> HttpResponse:
+    user = request.user
+    try:
```
Comment (rgasper) [inline caim_base/views/fosterer_profile.py]: is there a reason not to use `django.shortcuts.get_or_404`

Also, why is the Awg id the same as the user id? that doesn't smell right

File: caim_base/views/fosterer_profile.py [hunk 3]
```diff
@@ -357,9 +367,60 @@ def edit(request, stage_id):
...
+
+    context = {
+        "fosterer": fosterer,
+        "animal_type_labels": animal_type_labels,
+        "category_of_animals_labels": category_of_animals_labels,
+        "behaviour_labels": behaviour_labels,
+        "experience_categories_labels": experience_categories_labels,
+    }
+
+    pdf_string = render_to_string("fosterer_profile/pdf.html", context, request)
```
Comment (rgasper) [inline caim_base/views/fosterer_profile.py:416]: next five lines are sliiiick

File: caim_base/static/css/normalize.css [hunk 4]
```diff
@@ -0,0 +1,351 @@
+/*! normalize.css v8.0.1 | MIT License | github.com/necolas/normalize.css */
```
Comment (rgasper) [inline caim_base/static/css/normalize.css:1]: is this file only intended for use in the pdfs or should we be pulling this into the stylesheets for all of our pages?

**Review by rgasper — COMMENTED** [review]
Looks great, I had some questions tho

File: caim_base/views/fosterer_profile.py (hunk 2 above)
Comment (rwwiv) [reply caim_base/views/fosterer_profile.py]: I agree it doesn't smell right, this needs a second look. I think I was operating from an incorrect assumption here. There is a user - awg map that should likely be used here.

File: caim_base/static/css/normalize.css (hunk 4 above)
Comment (rwwiv) [reply caim_base/static/css/normalize.css:1]: That's a good question, I believe Bootstrap includes css normalization already so we shouldn't need it anywhere we're using that.

File: caim/settings.py (hunk 1 above)
Comment (rwwiv) [reply caim/settings.py:36]: I'm not sure, I don't believe so? I'm actually not sure why this variable wasn't already declared in our settings though to be honest.

**Review by genuineBuildMonkey — APPROVED** [review]
Looks good. Nice find on Weazyprint.

File: caim_base/views/fosterer_profile.py (hunk 2 above)
Comment (rwwiv) [reply caim_base/views/fosterer_profile.py]: Regarding `get_or_404()`, I think it does make sense to use this for the "fosterer" lookup.

**Pull request merged**